math/fastor/include/algebra/math/impl/fastor_matrix.hpp

```diff
@@ -0,0 +1,123 @@
+/** Algebra plugins, part of the ACTS project
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "algebra/math/common.hpp"
```

Comment:
I also don't think you need this header here. However, you are using the ```matrix_type```, which is defined in the storage header, I believe. Could you please include it here?

Reply:
No, its fine, don't worry about refactoring the other plugins :)